### DISCUSSION TITLE
Docker image with PDF signing

### DISCUSSION THREAD

**Discussion by johnt-softclouds** [Keyfactor/signserver-ce#35] [General]
Is there a SignServer-CE Docker image with the PDF signing plugin already installed?  If the answer is no, are there instructions on how to install the plugin using the SignServer-CE Docker image? Thank you.

**Comment by netmackan** [comment]
Yes, the official SignServer CE container image contains the PDF Signer (standard PDF signing not PAdES).
It is available on Docker Hub: https://hub.docker.com/r/keyfactor/signserver-ce

**Reply by binaryme** [reply]
Is it possible to sign with PAdES over SignServer CE?

**Reply by netmackan** [reply]
> Is it possible to sign with PAdES over SignServer CE?

No the PDF Signer does not support PAdES.
There is a different signer "AdES Signer" in enterprise edition which used a different implementation that supports both XAdES and PAdES but it is currently not available in Community.

**Comment by netmackan** [comment]
(I will move this issue to Discussions instead at that is where we discuss questions)
